src/styles/blocks/museum-hours.scss

```diff
@@ -0,0 +1,68 @@
+.museum-hours {
```

Comment:
you have a horizontal scroll because of this section, I suppose 😉

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/55802630/174538489-32379f5e-ec57-4840-b544-36df674bcabc.png">

Reply:
OMG I didn't notice it 😅🥹